src/Microsoft.VisualBasic/src/Microsoft/VisualBasic/Interaction.vb

```diff
@@ -1,21 +1,19 @@
 ' Licensed to the .NET Foundation under one or more agreements.
 ' The .NET Foundation licenses this file to you under the MIT license.
 ' See the LICENSE file in the project root for more information.
+Option Explicit On
+Option Strict On
 
 Namespace Microsoft.VisualBasic
 
-    Friend Module Interaction
-
+    Partial Public Module Interaction
         Friend Function IIf(Of T)(ByVal condition As Boolean, ByVal truePart As T, ByVal falsePart As T) As T
             If condition Then
                 Return truePart
             End If
 
             Return falsePart
         End Function
-
-
     End Module
-
 End Namespace
```

Comment:
Please revert changes to Interaction.vb if not needed in this PR.

Reply:
It is identical to commit 74126c by Omar, if you are asking me to do something with GIT, I need some instruction I see the revert command but I don't see if that reverts everything between the commits or only things to this one file.